<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreWindow</class>
 <widget class="QMainWindow" name="CoreWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="train_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load and Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="app_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="record_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Record and Test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
